Dockerfile

```diff
@@ -27,6 +25,7 @@ RUN make all
 
 FROM alpine:3.20 AS amf
 
+LABEL maintainer="Aether SD-Core <dev@lists.aetherproject.org>"
```

Comment:
Could you pls send test mail on this.

Reply:
Done. I just sent an email there. One caveat is that you need to subscribe to send/receive emails

Reply:
Is that information captured somewhere? How to subscriber for mailer?

Reply:
I do not see that being documented anywhere. As a matter of fact, I just signed up to the mailing list